Final Project Proposal

```diff
@@ -0,0 +1,36 @@
+Proposal of Final Project  Sheqi
+
+My plan is to write a program that can calculate the frequency of the words appearing in a document.
```

Comment:
This sounds like a great idea! Just to make sure, what functions do you think you would need to write in this program?